src/model/assignments.model.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  CreateDateColumn,
+  UpdateDateColumn,
+  DeleteDateColumn,
+  ManyToOne,
+  JoinColumn,
+} from 'typeorm'
+import Course from './courses.model'
+
+@Entity('assignments')
+export default class Assignment {
+  @PrimaryGeneratedColumn()
+  id: number
+
+  @ManyToOne(() => Course, course => course.id)
+
+  @JoinColumn({ name: 'course_id' })
+  courseId: number
```

Comment:
Just looked into this for the other PR, FK constraints the way we want them should look like 

```
  @Column({ name: 'course_id' })
  @JoinColumn({ name: 'course_id' })
  @ManyToOne(() => CourseModel)
  courseId: number
```

which admittedly is pretty damn close to what you've got here 👍  but we might want to consolidate your lines here such that it's clear that these things are related